### PR TITLE
bpo-42794: Update test_nntplib to use offical group name for testing

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue42794

### CODE REVIEW

**Comment by corona10** [comment]
I add the backport label for 3.7 and 3.6 also.
We often send backport patches for security issues but this issue breaks 3.7 and 3.6 testing also.
If there are other opinions please let me know.

**Comment by malemburg** [comment]
Patch looks good.

File: Lib/test/test_nntplib.py [hunk 1]
```diff
@@ -343,8 +344,9 @@ class NetworkedNNTP_SSLTests(NetworkedNNTPTests):
     # 400 connections per day are accepted from each IP address."
 
     NNTP_HOST = 'nntp.aioe.org'
-    GROUP_NAME = 'comp.lang.python'
-    GROUP_PAT = 'comp.lang.*'
+    GROUP_NAME = 'aioe.test'
```
Comment (malemburg) [inline Lib/test/test_nntplib.py:349]: It's probably a good idea to add a comment to this section, mentioning that aioe.test is one of the official groups on this server used for testing: https://news.aioe.org/manual/aioe-hierarchy/

Comment (vstinner) [reply Lib/test/test_nntplib.py:349]: If you add a comment, add "bpo-42794: " which always helps for future readers ;-)

**Review by vstinner — APPROVED** [review]
LGTM.

**Pull request merged**